README.md

```diff
@@ -1,34 +1,29 @@
-NU Hacks
-=======
+# hacks.github.io
 
-The website for NUHacks, a Northeastern University student organization for hackers and makers.
-
-## Local Server
-
-```bash
-# 1. build image
-docker build -t nuhacks $PWD
-# 2. start container for the first time (use `-it` instead of `-p` for process to ocupy terminal)
-docker run --rm -it -p 4000:4000 --name nuhacks -v $PWD:/srv/jekyll nuhacks
+## Project setup
+```
+yarn install
```

Comment:
- `package-lock.json` found.
- In `.gitignore` you have `yarn.lock` blacklisted

Are you sure you are using yarn?